Description: Disable sphinx.ext.intersphinx
 This avoids network access during the build of the docs.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-03-18

--- python-pbr-0.6.orig/doc/source/conf.py
+++ python-pbr-0.6/doc/source/conf.py
@@ -8,7 +8,7 @@ sys.path.insert(0, os.path.abspath('../.
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx']
+extensions = ['sphinx.ext.autodoc']
 
 # autodoc generation is a bit aggressive and a nuisance when doing heavy
 # text edit cycles.
